cts/lab/cts.in

```diff
@@ -8,6 +8,9 @@
 # or later (GPLv2+) WITHOUT ANY WARRANTY.
 #
 
+# e.g. /etc/sysconfig or /etc/default
+CONFIG_DIR=@CONFIGDIR@
+
```

Comment:
either like this or use @CONFIGDIR@ directly wherever needed.
slight benefit of the latter is that it is well known and one doesn't have to check back what CONFIG_DIR is
defined as.
but short simple file so either should be fine I guess.

Reply:
I like to do this with all configure variables so the replaceable parts are all grouped together at the top -- makes reading/reviewing the rest of the code easier

Reply:
depends on how you personally weight the pros & cons ;-)